src/Microsoft.SymbolStore/SymbolStores/HttpSymbolStore.cs

```diff
@@ -41,16 +41,21 @@ public HttpSymbolStore(ITracer tracer, SymbolStore backingStore, Uri symbolServe
             {
                 throw new ArgumentException(nameof(symbolServerUri));
             }
+
             // Normal unauthenticated client
-            _client = new HttpClient();
+            _client = new HttpClient {
+                Timeout = TimeSpan.FromMinutes(4)
```

Comment:
Can you give some background on this? Naively it seems a little odd that we would specify a non-default timeout, or that we would pick a timeout that was so long.

Reply:
The issue in the PR description (#87) has more details from the customer. In his case, the default timeout (100 secs) timed out on a long file download on his machine in a docker container.  I basically doubled the timeout and it fixed his problem. I don't have a sense of any downside, but the docs on this property don't give a lot of detail.

Reply:
My only worry would be if downloading symbols from an unresponsive server became really painful, for example downloading 50 files where each of them waits 4 minutes and then gives up. Given that this is only double the previous I'll assume if this was an issue people would have already complained. I'm fine leaving this as-is and then in the future if someone complains we'll think harder about how to address it.

Reply:
(and whoops, your description had reasonable info and I just missed it! My bad)